src/rrt/2dplane/PlaneStateSpace.hpp

```diff
@@ -18,6 +24,39 @@ class PlaneStateSpace : public StateSpace<POINT_CLASS> {
         return POINT_CLASS(drand48() * width(), drand48() * height());
     }
 
+    /**
+     * Generates a point based on a goal bias and a goal state
+     *
+     * @params goalState - Point that we want to extend towards
+     * @params goalBias - Changes likelihood of choosing a point near goal state
+     * @returns POINT_CLASS - random point based on goal bias and goal state
+     */
+    POINT_CLASS randomBiasState(const POINT_CLASS& goalState, float goalBias) const {
+        // Generates random value based on goalBias
+        float logitX = logit(goalBias, drand48() * .8 + .1);
```

Comment:
It might be better to use the c++ `rand` function here
see this: http://en.cppreference.com/w/cpp/numeric/random/rand

You can get a double from it by dividing it with `RAND_MAX`. I'm not sure if this random value has special significance though.